frontend/eda/Resources/projects/EditProject.tsx

```diff
@@ -44,13 +44,17 @@ function ProjectCreateInputs() {
         isReadOnly={true}
         label={t('SCM type')}
         placeholder={t('Git')}
+        labelHelp={t('There is currently only one SCM available for use.')}
       />
       <PageFormTextInput<EdaProject>
         name="url"
         isRequired={true}
         label={t('SCM URL')}
         placeholder={t('Enter SCM URL')}
         validate={isValidUrl}
+        labelHelp={t(
+          'A URL to a remote archive, such as Github or a build artifact stored in Artifactory and unpacks it into the project path for use.'
```

Comment:
This help text is not very clear, specifically the second part. Maybe we can split this into two sentences. Can we run it by UXD again? (Totally okay IMO to iterate and address in a follow-on PR if needed)